genesis.json

```diff
@@ -3917,18 +3917,18 @@
         "quorum": "0.3300000000",
         "threshold": "0.5000000000",
         "veto": "0.3340000000",
-        "governance_penalty": "0.0100000000"
+        "governance_penalty": "0.0000000000"
       }
     },
     "slashing": {
       "params": {
         "max-evidence-age": "86400000000000",
         "signed-blocks-window": "5000",
-        "min-signed-per-window": "0.5000000000",
+        "min-signed-per-window": "0.0500000000",
         "double-sign-unbond-duration": "300000000000",
-        "downtime-unbond-duration": "600000000000",
+        "downtime-unbond-duration": "0",
         "slash-fraction-double-sign": "0.2000000000",
-        "slash-fraction-downtime": "0.0500000000"
+        "slash-fraction-downtime": "0.0000000000"
```

Comment:
@jaekwon I see that you made these changes because the uptime condition (precommits) can easily be influenced by an attacker (especially with 1/3+), however even with a `slashing-fraction-downtime` of 0, where you could theoretically unjail immediately, an attacker can easily censor these transactions yielding the same result as with the original values.

Additionally, in a scenario where there is no attacker, these values completely remove the incentive for being online  (except the proposer bonus). Since jailing happens in the BeginBlocker right after the distribution, a validator can submit an Unjail transaction in exactly the block he would be jailed in and will thereby never be removed from the validatorset or miss any fee distribution tick as he is readded within the state commit.

Reply:
In Game of Stakes, we will incentivize liveness over self-delegation, total-delegation, or total stake.  So by the rules, it is already the case that liveness is incentivized.

In the live net, even without downtime penalty, the fact that such a validator is not earning inflationary rewards, should be sufficient for delegators to take action.

If there is a +1/3 cabal that is censoring one or more validators, and given +1/3 (example) that is *not* trying to censor the same validator, we will see this as either proposer blocks not being committed, so on the blockchain in the canonical commits (e.g. a cabal of proposers proposing successfully more often than others while others get censored, as you pointed out in another conversation), and/or, the live consensus messages for each validator (and depending on the network, the full nodes).

In the long-run, we make Tendermint liveness accountable by ultimately making all the p2p messages accountable, whether that means each validator signing WAL files and publishing these hash+signatures in a timely way.  But we're not there yet.

Meanwhile, for the hub, it will be up to the delegators and validators to be intelligent enough to identify attempts at censorship and delegate and defend against censoring cabals.  Ultimately it is detrimental to the network for there to be pro-censor and anti-censor cabals that degrade the performance of the network, so there is incentive for cooperation.

And until we learn more about the details of analysis tooling necessary, and build out more accountability tooling, it is more important that honest validators have a chance to re-try participation in the face of censorship, and to allow that, it is necessary to remove punishment for liveness failure than what already exists -- lack of ability to earn inflationary atoms and transaction fees.  Otherwise, the censorship cabal prevents honest validators from even trying to participate.  This change levels the playing field.

As @rigelrozanski pointed out, another option is to remove jailing altogether, and to let delegators more responsible for making the determination as to who ought to remain the validators, and risk complete liveness failure, and to punish the non-live validators then after making the appropriate determination as to who is responsible.